BUILD.gn

```diff
@@ -139,6 +143,20 @@ run_node("bundle") {
   ]
 }
 
+source_set("libdeno_nosnapshot") {
+  sources = [
+    "src/from_filesystem.cc",
+  ]
+  deps = [
+    ":bundle",
+    ":deno_bindings",
+  ]
+  configs += [ ":deno_config" ]
+  bt = get_target_outputs(":bundle")
+  bl = rebase_path(bt[0], root_build_dir)
```

Comment:
If you use `rebase_path(bt[0])` instead, it will set the `BUNDLE_LOCATION` to be an absolute path, which I think is better.

Also please use more descriptive variable names here `bundle_outputs` and `bundle_location`